### PR TITLE
Edit/542 page content

### PR DESCRIPTION
**Pull request by Erin-Cecele**
Thank you for contributing to the Quire Documentation & Website! Please complete the form below to submit your pull request for review.

*For the Title of this pull request, please use the format "Type/Issue-#: Brief description." For Type, the options are Edit, Add, Translate. Issue-# is only needed if this pull request addresses an existing issue.*

### Checklist 

Please put an X within the brackets that apply `[X]`.

- [X] I have read the [CONTRIBUTING.md](https://github.com/thegetty/quire/blob/main/CONTRIBUTING.md) file.

- [X] I have made my changes in a new branch and not directly in the main branch

- [X] I am requesting feedback on a draft pull request

### Is this pull request related to an open issue? If so, what is the issue number?

#542 

### Please describe the goal of this pull request and the changes that were made.

Update for 11ty

### Additional Comments

@geealbers a few questions: 

1. I think the HTML styling section could be explained better. Is there somewhere where you define the class before adding that wrapper? 
2. Will we still be using `hrefTargetBlank` in `config.yaml` to make links open in separate tab? For now, I've removed that mention.

### CODE REVIEW

File: content/docs-v1/page-content.md [hunk 1]
```diff
@@ -24,37 +25,42 @@ You can read all about Markdown syntax and how it is used in Quire in the [*YAML
...
-- Quire includes one pre-defined class called "backmatter". This is typically used to wrap bibliographies, appendices, and other related content at the end of an article or page, and will style them to match the default footnote styling.
 
-    ```go
-    {{</* q-class "backmatter" */>}} ...
-    {{</* /q-class */>}}
-    ```
+- There is an additional shortcode that is use to format backmatter. You can use the pairing of `{% backmatter %}` and `{% endbackmatter %}` to wrap bibliographies, appendices, and other related content at the end of an article or page. Whatever falls inside those two shortcodes will match the default footnote styling.
+
+```md
+{% backmatter %}
```
Comment (geealbers) [inline content/docs-v1/page-content.md:47]: Can you test this? You might need line breaks for it to work properly.

```md
{% backmatter %}

## Notes

{% endbackmatter %}
```

File: content/docs-v1/page-content.md [hunk 2]
```diff
@@ -24,37 +25,42 @@ You can read all about Markdown syntax and how it is used in Quire in the [*YAML
...
-- [`q-figure`](/docs-v1/figure-images/): Inserts a formatted figure image (including audio and video) and caption using data from the project’s `figures.yaml` file, or from values supplied directly in the shortcode.
-- [`q-figure-group`](/docs-v1/figure-images/): Like `q-figure`, but with handling for multiple images at once.
+## Use HTML to Add Styles
+
+You can also use HTML markup to wrap a block of text and apply certain styles to that text. The example below adds the class “alert” to the the phrase “Text goes here”. This can be useful in facilitating additional [style customizations](/docs-v1/styles-customization).
+
+```md
+<div class="alert">
+Text goes here
+</div>
```
Comment (geealbers) [inline content/docs-v1/page-content.md:62]: Line space before `</div>`

File: content/docs-v1/page-content.md [hunk 3]
```diff
@@ -104,37 +108,33 @@ An ID and the # symbol is also used for other kinds of elements on the same page
...
-See [note 3, chapter 2](/chapter-2/#fn:3)
+[referencetolink](#fn#)
+Also in regards to [note 21](#fn21)
+See [note 3, chapter 2](/chapter-2/#fn3)
 ```
 
 #### Citations
 
-When the citation shortcode `{{</* q-cite "author date" "page #" */>}}` is used in a body of text and corresponds to the short and full bibliographic information provided in the `references.yaml` file, an in-page bibliography will be generated and linked to. This linking is completed automatically.
+When the citation shortcode `{% cite "author date" "page #" %}` is used in a body of text and corresponds to the short and full bibliographic information provided in the `references.yaml` file, an in-page bibliography will be generated and linked to. This linking is completed automatically.
```
Comment (geealbers) [inline content/docs-v1/page-content.md:137]: We didn't have this complete in the Hugo version of the docs. The `cite` shortcode accepts three values, not two: `{% cite "author date" "page #" "display text" %}`

Semi-side-note: Matthew et al. generally use single straight quotes in shortcodes and I actually think it looks slightly cleaner: `{% cite 'author date' 'page #' 'display text' %}`. What do you think?

If it is better, we should be consistent both in the docs and in the starter kit whenever we clean it up

**Review by geealbers — CHANGES_REQUESTED** [review]

File: content/docs-v1/page-content.md [hunk 4]
```diff
@@ -24,37 +25,42 @@ You can read all about Markdown syntax and how it is used in Quire in the [*YAML
...
 
-While most Quire shortcodes work like `q-figure` as a single instance, the `q-class` shortcode acts as wrapper around other text and so it appears as a paired opening and closing shortcode. The closing code has a slash `/` preceding the shortcode name, much like you’d find in HTML markup. This example adds the class "alert" to the the phrase "Text goes here", which could be used to facilitate [style customization](/docs-v1/styles-customization/).
+The following shortcodes are currently available in Quire. You’ll find more about them in their respective sections of the guide, as well as in the [shortcodes API reference](/docs-v1/for-developers/#shortcodes-api).
 
-```go
-{{</* q-class "alert" */>}}
-Text goes here
-{{</* /q-class */>}}
-```
+- [`{% bibliography %}`](/docs-v1/citation-bibliographies/): Generates a bibliography from the entries in the project's `bibliography.yaml` file.
```
[the remark this answers is not in the document]
Comment (Erin-Cecele) [reply content/docs-v1/page-content.md:36]: ok yes, I'll go back and look at pages.md and make this correction.

File: content/docs-v1/page-content.md (hunk 1 above)
Comment (Erin-Cecele) [reply content/docs-v1/page-content.md:47]: Ah yes, I just discovered this with Chloe. The line breaks are necessary

File: content/docs-v1/page-content.md (hunk 3 above)
Comment (Erin-Cecele) [reply content/docs-v1/page-content.md:137]: ah, ok. I was wondering about that. It does look cleaner. I'll just need to go back and fix in various places. 

I will add the info about the three citation shortcode values to the citation section of the docs.

**Comment by Erin-Cecele** [comment]
@geealbers I have made the requested changes and a few other edits.

**Review by geealbers — APPROVED** [review]

**Pull request merged**